programmers/난이도별/level01.서울에서_김서방_찾기/HyeonJeong.java

```diff
@@ -0,0 +1,18 @@
+package level01.서울에서_김서방_찾기;
+
+public class HyeonJeong {
+    class Solution {
+        public String solution(String[] seoul) {
+            String answer = "";
+            for(int i = 0; i < seoul.length; i++)
+                if (seoul [i] == "Kim")
+                    answer = "김서방은 i에 있다";
```

Comment:
이렇게 적으면 i가 int i가 아닌 그냥 문자열 i로 인식해요!
```suggestion
                    answer = "김서방은 " + i + "에 있다";
```
이렇게 하면 제대로 나올 것 같습니다! 🙂